include/termcolor/termcolor.hpp

```diff
@@ -136,7 +136,8 @@ namespace termcolor
         #if defined(TERMCOLOR_OS_MACOS) || defined(TERMCOLOR_OS_LINUX)
             stream << "\033[4m";
         #elif defined(TERMCOLOR_OS_WINDOWS)
-        #endif
+			_internal::win_change_attributes(stream, -1, COMMON_LVB_UNDERSCORE);
+		#endif
```

Comment:
@trailfrenzy Thanks for the patch! Can you please fix the indentation? This project is formatted using spaces, not tabs.

Reply:
No problem.  I will make the correction.